src/app/keyboard/services/tv2-key-alias.service.ts

```diff
@@ -0,0 +1,25 @@
+import { KeyAliasService } from '../abstractions/key-alias-service'
+
+export class Tv2KeyAliasService implements KeyAliasService {
+  public getAliasesForKey(key: string): string[] {
+    switch (key) {
+      case 'AnyEnter':
```

Comment:
Should we have enums for this or is it overkill?

Reply:
I thought about it, and I think it is a good idea, but currently we have the rule about enum values to be uppercased snakecase and that enum keys should be the same as the values, which makes it impossible for this.

Reply:
We could make an exception for that rule?

Reply:
I'll do it, but I'll base it on the virtual keyboard branch.